lib/pitchfork/version.rb

```diff
@@ -2,7 +2,4 @@
 
 module Pitchfork
   VERSION = "0.13.0"
-  module Const
-    UNICORN_VERSION = '6.1.0'
```

Comment:
I'd like to keep that. As to claim some level of compatibility with Unicorn 6.1.